Add `4 Digit World Cup 2024 Concludes` news post (#11627)
ppy/osu-wiki: news/2024/2024-06-24-4-digit-world-cup-2024-concludes.md

Both teams' first picks turned out to be great choices leading to a 1-1 scoreline. Spain's second pick, however, didn't go as planned and Chile took away a breakpoint on NM2. They looked to consolidate the point with NM5, a low AR rhythm map, which on paper heavily favored the Chliean roster. Spain, despite having banned the other low AR map of the round, managed to put up a great team performance and took a breakpoint right back tying the match. After Spain won their next pick and went up ahead 2-3, a NM4 pick from Chile went down to the wire, and a great performance from M A N O L O sealed yet another breakpoint for Spain with a narrow 40,000 score gap, bringing them to match point. For Spain's last pick, they secured the win on DT1 with another relatively close lead of 150,000, knocking the 4th seed Chile down to the lower bracket and advancing to face 21st seed Taiwan in the Round of 16. Taiwan had also pulled an incredible upset in their match, defeating the 12th seed Japan.

### Semifinals: ::{ flag=KR }:: South Korea vs. ::{ flag=CN }:: China

### Grand Finals ::{ flag=US }:: United States vs. ::{ flag=CA }:: Canada

> ## Interviews

Reviewer: all the longer paragraphs should be split in half

::{#blanktap}::

[![](/wiki/shared/news/2024-06-24-4-digit-world-cup-2024-concludes/blanktap-banner.png)](https://osu.ppy.sh/users/10137131)

> Hello, I'm BlankTap. I'm a bit of a tournament player and recently more known as a commentator, probably one that gets a bit too excited at times. I've been commentating actively for one and half years now and had the chance to contribute regarding 4WC's casting.

**BlankTap** has been making a name for himself in recent years thanks to his high-energy commentary, and he was an integral part of the casting team during this year's 4WC.

---

*What were your thoughts participating in your first world-cup-style tournament as a commentator? How did you prepare yourself?*

> I was very excited and basically in disbelief. The overturning of what's initially a rejection of my commentating application energized me up to prove that this will not be a decision to be regretted upon. I really do want to say thank you and that I am eternally indebted to those who believed in me that made this possible.
> 

> > Regarding preparation, I don't think I can explain much other than that I basically did nearly nothing. I've only discovered the archetypes of commentating recently and while this wouldn't fly when you're an analyst or a color caster, play by play casting needs much less intricate knowledge of a team's strategy or skillset and rather how the current map will go in which I do play some of the mappool for fun and I do have some knowledge regarding how a map generally would go for various skillset from the experience I've accumulated as a tournament player myself. In short, my preparation consists of filling up my water bottle, sitting comfortably in my chair, and going hard when the moment needs it to be.

Reviewer: second sentence is very long-winded

*This year's 4WC had a lot of upsets in the earlier rounds. Do you believe that this is something that could happen more frequently from now on and does it make commentating more exciting for you?*

> > Regarding the plentiful early round upsets that were witnessed in this iteration, it really has to do with how the earlier rounds consists of lower star rating maps and the big focus on a player's consistency rather than their skillcap. I do think that every country is as good as each other up until a certain star rating. There are no safe rounds in a world cup where every nation sends in their best of the best. High seeding is nothing more but an intimidation factor if you don't back it up.

Reviewer: ```suggestion
> Regarding the plentiful early round upsets that were witnessed in this iteration, it really has to do with how the earlier rounds consist of lower star rating maps and the big focus on a player's consistency rather than their skillcap. I do think that every country is as good as each other up until a certain star rating. There are no safe rounds in a world cup where every nation sends in their best of the best. High seeding is nothing more but an intimidation factor if you don't back it up.
```

> 

> > Nonetheless, teams do get high seedings not without a merit. I don't think upsets are something that will happen more often but only as often as or less than it is now. If upsets do spike up in quantity, it is only because of an erroneous early round pooling that deviates completely from the qualifier (Ex. speed qualifier into full gimmick RO32). Of course, its frequency is not what makes an upset exciting. We all love an underdog story, the David against the Goliath, this is no exception especially when it comes to a tiebreaker. At that point, the audience and the casters have won regardless of the result.

Reviewer: ```suggestion
> Nonetheless, teams do get high seedings not without a merit. I don't think upsets are something that will happen more often but only as often as or less than it is now. If upsets do spike up in quantity, it is only because of an erroneous early round pooling that deviates completely from the qualifier (Ex. speed qualifier into full gimmick RO32). Of course, its frequency is not what makes an upset exciting. We all love an underdog story, the David against the Goliath, and this is no exception especially when it comes to a tiebreaker. At that point, the audience and the casters have won regardless of the result.
```

> 
> I'm Shurelia, a catch player (despite being involved in mapping for standard tournaments) since 2012 and also a GMT since 2014. I've been actively mapping for standard since 2014 and finally got into tournament mapping in 2023.

**Shurelia** has started, despite being an osu!catch player, making tournament maps for osu! recently, including a total of three for this year's 4WC.

---

*What brought you into mapping? Why have you decided to create custom maps for tournaments such as 4WC 2024?*

> The reason why I started mapping for osu! standard specifically was due to being a heavy convert player in catch and wanting to gain #1 on a song that I really like. But the reason why I WANT to be better in mapping is because I got captivated by [Lan Wing](https://osu.ppy.sh/users/467860)'s beatmap of [t=NODE - Four Seasons](https://osu.ppy.sh/beatmapsets/285718#osu/645242). This map moves me so much that I randomly shed tears despite having no clue about osu! standard at all due to how beautiful the map and the song is.
> 

> > I decided to create custom maps because I just love watching people playing my maps, moreover when it's in a big tournament such as 4WC 2024. It also lets me learn many things to improve my mapping knowledge gameplay wise by learning what the players think about my map when they try it, because as I mentioned, I do not play osu! standard at all and thus it's obvious that I don't testplay my own maps at all. Custom mapping is such a good way to improve myself, at least that's what I'm thinking.

Reviewer: wrong word

```suggestion
> I decided to create custom maps because I just love watching people playing my maps, especially when it's in a big tournament such as 4WC 2024. It also lets me learn many things to improve my mapping knowledge gameplay wise by learning what the players think about my map when they try it, because as I mentioned, I do not play osu! standard at all and thus it's obvious that I don't testplay my own maps at all. Custom mapping is such a good way to improve myself, at least that's what I'm thinking.
```

> It feels great! Unfortunately, due to my circumstances, my interest with osu! standard tournaments isn't as high as people might think and thus I only watch a match if my map got picked so I can see how the chat or the players reacted to it.

## Statistics

A lot happened this tournament that can be shown off by statistics! We've compiled all of the per-round statistics into a [neat folder](https://drive.google.com/drive/u/0/folders/1dvBabKdU7V1nx1wEMhINi89NSdBvDU5f), but here are some extra picks that deserve their own mentions:

- There were a total of 33 scores set worth more than 500pp, the highest of which being [scylla](https://osu.ppy.sh/users/9405745)'s 651pp full combo on [BABYMETAL - Arkadia \[Extreme\]](https://osu.ppy.sh/beatmapsets/1695872#osu/3644487) in Sweden's [Semifinals match against Brazil](https://osu.ppy.sh/community/matches/114120011).
- We saw a total of 16 seed upsets out of 62 matches played, the biggest of which were:
  - [Chile (5) vs. Spain (28)](https://osu.ppy.sh/community/matches/113860341) in the Round of 32, in which 98% of pick'em players incorrectly predicted Chile to win.

>   - [South Korea (2) vs. China (14)](https://osu.ppy.sh/community/matches/114131751) in the Semifinals, notable for being very late in the bracket to have such an upset.

Reviewer: vod links instead of mp links?

Author: prolly better yeah, will update once they're all uploaded (later today)

This year's 4WC has seen many changes compared to the previous iteration to stay up to date in the everchanging nature of osu! tournaments. This includes a more consistent difficulty scaling, an increase in custom beatmaps, a larger involvement with the playerbase as well as taking in new and talented staff members to help them flourish and elevate the 4 Digit World Cup to newer heights with their sparkling passion and creativity!

## Summary

![](/wiki/Tournaments/4WC/2024/img/podium.jpg)

Congratulations to the United States for making 4WC history by being the first claim the title of champions twice in a row, as well as being the first team to garner three total 4WC wins. The United States had their strongest showing to date in this iteration of the tournament, only losing 10 points total in their 6 matches. Canada put in a strong effort, but there was never a moment where the United States was on the backfoot, and with half of their team being new to the roster, their depth remains unmatched.

> As for our other podium contenders, we have Canada and China, whose roads have been intertwined since the beginning of this tournament. They faced off in RO16 first, where China eked out a 5–3 victory for a massive upset against the 3rd seed, which was only foreshadowing for the future. China then went on to, as the 14th seed, dominate the 6th seed Brazil and have an all out war with the 2nd seed South Korea, being able to consolidate after 5 breakpoints in a row. This was their first time on the podium, and a massive improvement from their placement the year prior. As for the Canadian side, the early loss to China meant that they had to play the most potential matches in losers bracket, winning 7 matches in a row to claim their #2 slot once more.

Reviewer: ```suggestion
As for our other podium contenders, we have Canada and China, whose roads have been intertwined since the beginning of this tournament. They faced off in RO16 first, where China eked out a 5–3 victory for a massive upset against the 3rd seed, which was only foreshadowing for the future. China then went on to, as the 14th seed, dominate the 6th seed Brazil and have an all out war with the 2nd seed South Korea, being able to consolidate after 5 breakpoints in a row. This was their first time on the podium, and a massive improvement from their placement the year prior. As for the Canadian side, the early loss to China meant that they had to play the most potential matches in losers' bracket, winning 7 matches in a row to claim their #2 slot once more.
```

> As for our other podium contenders, we have Canada and China, whose roads have been intertwined since the beginning of this tournament. They faced off in RO16 first, where China eked out a 5–3 victory for a massive upset against the 3rd seed, which was only foreshadowing for the future. China then went on to, as the 14th seed, dominate the 6th seed Brazil and have an all out war with the 2nd seed South Korea, being able to consolidate after 5 breakpoints in a row. This was their first time on the podium, and a massive improvement from their placement the year prior. As for the Canadian side, the early loss to China meant that they had to play the most potential matches in losers' bracket, winning 7 matches in a row to claim their #2 slot once more.

Reviewer: ```suggestion
As for our other podium contenders, we have Canada and China, whose roads have been intertwined since the beginning of this tournament. They faced off in RO16 first, where China eked out a 5–3 victory for a massive upset against the 3rd seed, which was only foreshadowing for the future.

China then went on to, as the 14th seed, dominate the 6th seed Brazil and have an all out war with the 2nd seed South Korea, being able to consolidate after 5 breakpoints in a row. This was their first time on the podium, and a massive improvement from their placement the year prior. As for the Canadian side, the early loss to China meant that they had to play the most potential matches in losers' bracket, winning 7 matches in a row to claim their #2 slot once more.
```

</div>

It is time for Semifinals! Many different countries had been predicted to possibly face off against South Korea on this stage, but it was China, seeded 14th, that turned the tournament on its head. Defying expectations, China had defeated the 3rd seed Canada in the Round of 16 and the 6th seed Brazil in the Quarterfinals, proving that they are a force to be reckoned with.

On the other side, the 2nd seed South Korea had advanced to the Semifinals with relative ease, losing only a few points along the way. The match between China and South Korea promised excitement, and it delivered in full.

The match began with South Korea securing a win on their first pick followed by a breakpoint, setting the tone for what seemed to be another dominant performance. However, China quickly responded with a breakpoint of their own on DT3, followed by a thrilling victory on NM5 despite a mid-map disconnect from a Chinese player, winning by a mere 3,000 points. They continued their momentum with another breakpoint on NM6, securing three consecutive points and a 3–2 lead.

> The teams then traded breakpoints again on the next two picks. China broke the chain with a win on their DT1 pick, bringing them to match point at 5–3. South Korea fought back with a strong performance on FM1, narrowing the gap to 5–4. The final pick, NM4, saw China clinching victory with a narrow 40,000 point lead. This remarkable win knocked another top seed, South Korea, down to the losers bracket, where they went on to face Canada. Meanwhile, China advanced to take on the United States in the Finals.

Reviewer: ```suggestion
The teams then traded breakpoints again on the next two picks. China broke the chain with a win on their DT1 pick, bringing them to match point at 5–3. South Korea fought back with a strong performance on FM1, narrowing the gap to 5–4. The final pick, NM4, saw China clinching victory with a narrow 40,000 point lead. This remarkable win knocked another top seed, South Korea, down to the losers' bracket, where they went on to face Canada. Meanwhile, China advanced to take on the United States in the Finals.
```

### Grand Finals ::{ flag=US }:: United States vs. ::{ flag=CA }:: Canada

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/2DlShXHRRwc" frameborder="0" allowfullscreen></iframe>
</div>

With a rematch of last year's Grand Finals, the Canadian team once again stands up to fight the United States for the top spot on the podium after a narrow 6–7 loss in 2023. Featuring the hardest mappool in the tournament series to date, this stage isn't trivially easy for anyone to perform on.

> Canada had a tough journey this year, dropping to the losers bracket early after a defeat to China in the Round of 16. They managed to put together an impressive comeback though, knocking out several formidable opponents including the 3rd seed South Korea as well as China, the very team that had sent them to the losers bracket earlier. Meanwhile, the United States maintained their dominance throughout, showing no major weaknesses on their way to the Grand Finals. The question now was: will this match be as close and thrilling as last year, or a more one-sided showing?

Reviewer: ```suggestion
Canada had a tough journey this year, dropping to the losers' bracket early after a defeat to China in the Round of 16. They managed to put together an impressive comeback though, knocking out several formidable opponents including the 3rd seed South Korea as well as China, the very team that had sent them to the losers' bracket earlier. Meanwhile, the United States maintained their dominance throughout, showing no major weaknesses on their way to the Grand Finals. The question now was: will this match be as close and thrilling as last year, or a more one-sided showing?
```

> 
> In the end, we still lost the hard-fought battle, but USA's paramount performance was absolutely deserving of the win. After the match, we were still in high spirits. We still knew that second place was something to be proud of and that we had an amazing run through the tournament.

*Losing in the Round of 16 against China surely was a massive hit. Have you had any doubts in yourself, your team and how did you manage to bounce back together with your team?*

> This year was an extremely competitive year for 4WC, with a majority of teams and players having major improvements since 4WC 2023. We knew that underestimating the bracket could result in an upset and Canada dropping down to the loser's bracket earlier than expected.
> 

> > With this in mind, Canada started the match against China by picking NM2: Yousei Teikoku - Zetsu \[LMT's Extra\], one of our least confident picks due to a core member oversleeping the match. However this was an oversight, as China's stellar performance led to us immediately losing a breakpoint. We knew that we were underprepared, and we paid for that.

Reviewer: ```suggestion
> With this in mind, Canada started the match against China by picking NM2: [Yousei Teikoku - Zetsu \[LMT's Extra\]](https://osu.ppy.sh/beatmapsets/1262446#osu/2929650), one of our least confident picks due to a core member oversleeping the match. However this was an oversight, as China's stellar performance led to us immediately losing a breakpoint. We knew that we were underprepared, and we paid for that.
```

As for our other podium contenders, we have Canada and China, whose roads have been intertwined since the beginning of this tournament. They faced off in RO16 first, where China eked out a 5–3 victory for a massive upset against the 3rd seed, which was only foreshadowing for the future.

China then went on to, as the 14th seed, dominate the 6th seed Brazil and have an all out war with the 2nd seed South Korea, being able to consolidate after 5 breakpoints in a row. This was their first time on the podium, and a massive improvement from their placement the year prior. As for the Canadian side, the early loss to China meant that they had to play the most potential matches in losers' bracket, winning 7 matches in a row to claim their #2 slot once more.

## Match highlights

### Round of 32: ::{ flag=CL }:: Chile vs. ::{ flag=ES }:: Spain

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/vYchqxY_2SI" frameborder="0" allowfullscreen></iframe>
</div>

> In most tournaments, the first-round matchups typically don't feature massive upsets or many close matches. However, in this year's 4WC, things heated up right from the start. There were numerous seed upsets left and right, especially on the first day of matches. The most exciting and surprising match certainly was Chile vs. Spain.

Reviewer 2: ```suggestion
In most tournaments, the first-round match-ups typically don't feature massive upsets or many close matches. However, in this year's 4WC, things heated up right from the start. There were numerous seed upsets left and right, especially on the first day of matches. The most exciting and surprising match certainly was Chile vs. Spain.
```

The teams then traded breakpoints again on the next two picks. China broke the chain with a win on their DT1 pick, bringing them to match point at 5–3. South Korea fought back with a strong performance on FM1, narrowing the gap to 5–4. The final pick, NM4, saw China clinching victory with a narrow 40,000 point lead. This remarkable win knocked another top seed, South Korea, down to the losers' bracket, where they went on to face Canada. Meanwhile, China advanced to take on the United States in the Finals.

### Grand Finals: ::{ flag=US }:: United States vs. ::{ flag=CA }:: Canada

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/2DlShXHRRwc" frameborder="0" allowfullscreen></iframe>
</div>

With a rematch of last year's Grand Finals, the Canadian team once again stands up to fight the United States for the top spot on the podium after a narrow 6–7 loss in 2023. Featuring the hardest mappool in the tournament series to date, this stage isn't trivially easy for anyone to perform on.

> Canada had a tough journey this year, dropping to the losers' bracket early after a defeat to China in the Round of 16. They managed to put together an impressive comeback though, knocking out several formidable opponents including the 3rd seed South Korea as well as China, the very team that had sent them to the losers' bracket earlier. Meanwhile, the United States maintained their dominance throughout, showing no major weaknesses on their way to the Grand Finals. The question now was: will this match be as close and thrilling as last year, or a more one-sided showing?

Reviewer 2: is this just me or is China being bundled together with South Korea in the same sentence with no separation (like a comma) just awkward? No tsure if a comma would work best or just rewording this.
`[...]knocking out several formidable opponents including the 3rd seed South Korea as well as China[...]`

Author: would `[...]South Korea and later China[...]` go better

Reviewer 2: yessir

---

*How did it feel to face off against the United States in Grand Finals, knowing that Canada had lost against them last year already?*

> Before going into Grand Finals, we knew that USA was going to be the team to beat for a spot at the top. The heartbreaking defeat on tiebreaker last year gave us hope that USA was a contestable, and beatable team. Knowing this, we put our best effort into preparing for the weekend. We knew that gimmick was going to be our crutch, and main focus in our picking strategy. One breakpoint could be the single deciding factor in the match, and any mistake could have a grave cost.
> 
> In the end, we still lost the hard-fought battle, but USA's paramount performance was absolutely deserving of the win. After the match, we were still in high spirits. We still knew that second place was something to be proud of and that we had an amazing run through the tournament.

*Losing in the Round of 16 against China surely was a massive hit. Have you had any doubts in yourself, your team and how did you manage to bounce back together with your team?*

> > This year was an extremely competitive year for 4WC, with a majority of teams and players having major improvements since 4WC 2023. We knew that underestimating the bracket could result in an upset and Canada dropping down to the loser's bracket earlier than expected.

Reviewer 2: ```suggestion
> This year was an extremely competitive year for 4WC, with a majority of teams and players having major improvements since 4WC 2023. We knew that underestimating the bracket could result in an upset and Canada dropping to the loser's bracket earlier than expected.
```

::{#blanktap}::

[![](/wiki/shared/news/2024-06-24-4-digit-world-cup-2024-concludes/blanktap-banner.png)](https://osu.ppy.sh/users/10137131)

> Hello, I'm BlankTap. I'm a bit of a tournament player and recently more known as a commentator, probably one that gets a bit too excited at times. I've been commentating actively for one and half years now and had the chance to contribute regarding 4WC's casting.

**BlankTap** has been making a name for himself in recent years thanks to his high-energy commentary, and he was an integral part of the casting team during this year's 4WC.

---

*What were your thoughts participating in your first world-cup-style tournament as a commentator? How did you prepare yourself?*

> I was very excited and basically in disbelief. The overturning of what's initially a rejection of my commentating application energized me up to prove that this will not be a decision to be regretted upon. I really do want to say thank you and that I am eternally indebted to those who believed in me that made this possible.
> 

> > Regarding preparation, I don't think I can explain much other than that I basically did nearly nothing. I've only discovered the archetypes of commentating recently and while this wouldn't fly when you're an analyst or a color caster, play by play casting needs much less intricate knowledge of a team's strategy or skillset. It revolves more around how the current map will go, where it helps that I do play parts of the mappools for fun and have some knowledge regarding how a map generally would go for from the experience I've accumulated as a tournament player myself.

Reviewer 2: ```suggestion
> Regarding preparation, I don't think I can explain much other than that I basically did nearly nothing. I've only discovered the archetypes of commentating recently and while this wouldn't fly when you're an analyst or a color caster, play by play casting needs much less intricate knowledge of a team's strategy or skill set. It revolves more around how the current map will go, where it helps that I do play parts of the mappools for fun and have some knowledge regarding how a map generally would go for from the experience I've accumulated as a tournament player myself.
```

> 
> In short, my preparation consists of filling up my water bottle, sitting comfortably in my chair, and going hard when the moment needs it to be.

*This year's 4WC had a lot of upsets in the earlier rounds. Do you believe that this is something that could happen more frequently from now on and does it make commentating more exciting for you?*

> > Regarding the plentiful early round upsets that were witnessed in this iteration, it really has to do with how the earlier rounds consist of lower star rating maps and the big focus on a player's consistency rather than their skillcap. I do think that every country is as good as each other up until a certain star rating. There are no safe rounds in a world cup where every nation sends in their best of the best. High seeding is nothing more but an intimidation factor if you don't back it up.

Reviewer 2: ```suggestion
> Regarding the plentiful early round upsets that were witnessed in this iteration, it really has to do with how the earlier rounds consist of lower star rating maps and the big focus on a player's consistency rather than their skill cap. I do think that every country is as good as each other up until a certain star rating. There are no safe rounds in a world cup where every nation sends in their best of the best. High seeding is nothing more but an intimidation factor if you don't back it up.
```

> 
> I'm Shurelia, a catch player (despite being involved in mapping for standard tournaments) since 2012 and also a GMT since 2014. I've been actively mapping for standard since 2014 and finally got into tournament mapping in 2023.

**Shurelia** has started, despite being an osu!catch player, making tournament maps for osu! recently, including a total of three for this year's 4WC.

---

*What brought you into mapping? Why have you decided to create custom maps for tournaments such as 4WC 2024?*

> The reason why I started mapping for osu! standard specifically was due to being a heavy convert player in catch and wanting to gain #1 on a song that I really like. But the reason why I WANT to be better in mapping is because I got captivated by [Lan Wing](https://osu.ppy.sh/users/467860)'s beatmap of [t=NODE - Four Seasons](https://osu.ppy.sh/beatmapsets/285718#osu/645242). This map moves me so much that I randomly shed tears despite having no clue about osu! standard at all due to how beautiful the map and the song is.
> 

> > I decided to create custom maps because I just love watching people playing my maps, especially when it's in a big tournament such as 4WC 2024. It also lets me learn many things to improve my mapping knowledge gameplay wise by learning what the players think about my map when they try it, because as I mentioned, I do not play osu! standard at all and thus it's obvious that I don't testplay my own maps at all. Custom mapping is such a good way to improve myself, at least that's what I'm thinking.

Reviewer 2: i think its fair to say that testing a map *is* playing it so not sure if the original wording is valid, this is kinda subjective though i dont have a strong opinion on this
```suggestion
> I decided to create custom maps because I just love watching people playing my maps, especially when it's in a big tournament such as 4WC 2024. It also lets me learn many things to improve my mapping knowledge gameplay wise by learning what the players think about my map when they try it, because as I mentioned, I do not play osu! standard at all and thus it's obvious that I don't test my own maps at all. Custom mapping is such a good way to improve myself, at least that's what I'm thinking.
```

Author: mapper friend said testplay makes it a lot more clear that its specifically *playing* and not something else so keeping as is

*How does it feel to have your maps be played in high-stake tournaments? Does it motivate a mapper to follow the tournament more closely or only if their map is played?*

> It feels great! Unfortunately, due to my circumstances, my interest with osu! standard tournaments isn't as high as people might think and thus I only watch a match if my map got picked so I can see how the chat or the players reacted to it.

## Statistics

A lot happened this tournament that can be shown off by statistics! We've compiled all of the per-round statistics into a [neat folder](https://drive.google.com/drive/u/0/folders/1dvBabKdU7V1nx1wEMhINi89NSdBvDU5f), but here are some extra picks that deserve their own mentions:

- There were a total of 33 scores set worth more than 500 pp, the highest of which being ::{ flag=SE }:: [scylla](https://osu.ppy.sh/users/9405745)'s 651 pp full combo on [BABYMETAL - Arkadia \[Extreme\]](https://osu.ppy.sh/beatmapsets/1695872#osu/3644487) in Sweden's [Semifinals match against Brazil](https://osu.ppy.sh/community/matches/114120011).
- We saw a total of 16 seed upsets out of 62 matches played, the biggest of which were:

>   - [Chile (5) vs. Spain (28)](https://www.youtube.com/watch?v=vYchqxY_2SI) in the Round of 32, in which 98% of pick'em players incorrectly predicted Chile to win.

Reviewer 2: IMO looks bad without the hash sign (subjective though)
```suggestion
  - [Chile (#5) vs. Spain (#28)](https://www.youtube.com/watch?v=vYchqxY_2SI) in the Round of 32, in which 98% of pick'em players incorrectly predicted Chile to win.
```

Author: should these also have the flags or does it mess up the links

Reviewer 2: hmm we can try i never put flags inside of hyperlinked text

Author: we shall attempt

>   - [South Korea (2) vs. China (14)](https://www.youtube.com/watch?v=UlJ0BW0XI4M) in the Semifinals, notable for being very late in the bracket to have such an upset.

Reviewer 2: ```suggestion
  - [South Korea (#2) vs. China (#14)](https://www.youtube.com/watch?v=UlJ0BW0XI4M) in the Semifinals, notable for being very late in the bracket to have such an upset.
```

> - Panama qualified for their first ever world-cup-style tournament with ::{ flag=PA }:: [Leonard H](https://osu.ppy.sh/users/11562243)'s (a.k.a. hammytr) #10 individual performance boosting the team to the 32nd spot.

Reviewer 2: this reads weird
```suggestion
- Panama qualified for their first ever world-cup-style tournament with ::{ flag=PA }:: [Leonard H](https://osu.ppy.sh/users/11562243) (a.k.a. hammytr) getting 10th place in individual performance statistics, boosting the team to the 32nd spot.
```
